libs/common/src/lib/policies/nursery.policy.ts

```diff
@@ -23,6 +24,7 @@ export class NurseryPolicy extends UserPermissionsPolicy {
         ];
         if (projectIds.length > 0) {
           this.builder.can("read", Nursery, { projectId: { $in: projectIds } });
+          this.builder.can("delete", Nursery, { projectId: { $in: projectIds } });
```

Comment:
According to the PHP nursery soft delete controller, a nursery may only be deleted by a non-admin if there have been no reports generated. I was hoping we could implement that restriction in the policy, but I'm not thinking of a clean way to do it, so I think instead the nursery processor needs to overload `delete` and perform the check there.